app/Jobs/PlatformStatsSummaryJob.php

```diff
@@ -2,10 +2,12 @@
 
 namespace App\Jobs;
 
+use App\Traits;
 use App\Helper\MWTimestampHelper;
 use App\Wiki;
 use App\User;
 use Illuminate\Database\DatabaseManager;
+use Illuminate\Support\Facades\Http;
```

Comment:
If I'm not mistaken this is not used anymore now that the code has been moved to the trait, but it also won't really hurt much.

Reply:
oh yes I missed that